core/templates/components/state-editor/state-hints-editor/state-hints-editor.component.ts

```diff
@@ -130,21 +129,16 @@ angular.module('oppia').component('stateHintsEditor', {
         AlertsService.clearWarnings();
         ExternalSaveService.onExternalSave.emit();
 
-        $uibModal.open({
-          template: require(
-            'pages/exploration-editor-page/editor-tab/templates/' +
-            'modal-templates/add-hint-modal.template.html'),
-          backdrop: 'static',
-          resolve: {},
-          windowClass: 'add-hint-modal',
-          controller: 'AddHintModalController'
+        NgbModal.open(AddHintModalComponent, {
+          backdrop: 'static'
         }).result.then(function(result) {
```

Comment:
Why did we take away the `windowClass`?

Reply:
That was removed by mistake, added it now.